src/algebraic_geometry/properties.lean

```diff
@@ -83,4 +83,49 @@ begin
     { intro x, exact x.rec _ } }
 end
 
+lemma has_generic_point [is_integral X] :
+  ∃ (x : X.carrier), closure ({x} : set X.carrier) = ⊤ :=
+begin
+  haveI : irreducible_space X.X := show irreducible_space X.carrier, by apply_instance,
+  obtain ⟨⟨U, hU⟩, R, ⟨e⟩⟩ := X.local_affine (nonempty.some infer_instance),
+  haveI : nonempty (U.open_embedding.is_open_map.functor.obj ⊤) := ⟨⟨_, ⟨_, hU⟩, trivial, rfl⟩⟩,
+  haveI := ((LocallyRingedSpace.Γ.map_iso e.op).symm ≪≫ Spec_Γ_identity.app R :
+    X.presheaf.obj (op $ U.open_embedding.is_open_map.functor.obj ⊤) ≅ R)
+      .CommRing_iso_to_ring_equiv.symm.is_domain _,
+  let e' : prime_spectrum R ≃ₜ U.1 :=
+    (homeo_of_iso $ LocallyRingedSpace.forget_to_Top.map_iso e).symm,
+  let x := e' ⟨0, ideal.bot_prime⟩,
+  use x.1,
+  rw closed_eq_top_iff_nonempty_open_subset U.2 (nonempty.intro ⟨_, hU⟩) is_closed_closure,
+  intros y hy,
+  have : closure ({⟨0, ideal.bot_prime⟩} : set $ prime_spectrum R) = ⊤ :=
+    eq_top_iff.mpr (λ _ _, (prime_spectrum.le_iff_mem_closure _ _).mp bot_le),
+  apply_fun (set.image e') at this,
+  rw [e'.image_closure, set.image_singleton] at this,
+  have h' := @closure_subtype _ _ U.1 ⟨y, hy⟩ ({x} : set U),
+  rw [set.image_singleton, subtype.coe_mk] at h',
+  rw [subtype.val_eq_coe, ← h', this, set.top_eq_univ, set.image_univ,
+    set.range_iff_surjective.mpr e'.surjective],
+  trivial
+end
+
+/-- The generic point of an integral scheme. -/
+noncomputable
+def Scheme.generic_point [is_integral X] : X.carrier := (has_generic_point X).some
```

Comment:
Would it be better to write out the explicit construction?

Reply:
I don't think so.
It would essentially be this part of the proof above, and I don't really think this is particularly useful / easy to work with. 
https://github.com/leanprover-community/mathlib/blob/7a90f61a5dd33cd022410f79839ee6df8ac65809/src/algebraic_geometry/properties.lean#L89-L98

We will still know that the zero ideal of an affine open is the generic point later on though, probably via the fact that dominant morphisms preserve generic points.